scripts/client/lint-rust.mjs

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env zx
-import 'zx/globals';
-import { getClippyToolchain, workingDirectory } from '../utils.mjs';
+import "zx/globals";
```

Comment:
nit: most of the files use single quotes -- is there a reason to change these?

Reply:
Something to do with my VSC prettier configuration most likely. I manually fixed them.